GVFS/GVFS.Common/MountInternal.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GVFS.Common
+{
+    public class MountInternal

```

Comment:
This currently sounds like a method name rather than a class name, and also these parameters apply to all verbs, not just mount. Rename to `InternalVerbParameters`?

Reply:
Also does it belong here in `GVFS.Common`? I think it could probably move into the CLI project instead.

Reply:
@sanoursa CLI project?  Sorry i'm not seeing it.  It's used by GVFS.Service/GVFS.Windows(Mac) which is why I was thinking Common might be a good home.

Reply:
`GVFS.Windows` and `GVFS.Mac` are what I meant by the CLI project(s), since they produce the CLI executable.

This class exists for interacting with the CLI, which is why I suggested putting it there. `GVFS.Service` should be able to take a dependency on `GVFS.Windows`.

Reply:
You're right, this is a class specifically for `MountVerb`.

It should also be used by whatever Mac equivalent of the `GVFS.Service` project will be. I imagine that the Mac service can take a dependence on `GVFS.Mac` and get the same functionality. Is that right?

Reply:
It's not specifically for `MountVerb`, it gets used by all CLI verbs.

And yes the Mac equivalent of the service should be able to depend on the CLI project as well.

Reply:
@sanoursa I tried to add a dependency on GVFS.Windows from GVFS.Service.

However since GVFS.Service has a dependency on GVFS.Service.UI which has a shared file ( PlatformLoader.Windows.cs), I'm getting build errors saying it doesn't know which GVFSPlatformLoader to use.  Not sure how to resolve this one.

Reply:
I tried removing the shared file, to no avail.  As you've previously mentioned GVFS.Service.UI needs some attention in terms of refactoring / getting it to work again.  I'm going to leave this final change out of scope for the PR.  But hopefully we prioritize the notifications soon and then we can clean this up.